src/pouch.js

```diff
@@ -123,6 +123,16 @@ class Pouch {
     })
   }
 
+  byRemoteIdMaybe (id, callback) {
+    this.byRemoteId(id, (err, was) => {
+      if (err && (err.status !== 404)) {
```

Comment:
I extracted this from [v2's RemoteWatcher](https://github.com/cozy-labs/cozy-desktop/blob/master/src/remote/v2/watcher.js#L171-L173).

I assumed this was some PouchDB behavior, but it's actually done in `byRemoteId` (line 119, just before this one).
Maybe I could have used `this.db.query` directly.